ECMA2Yaml/ECMAHelper/Models/ECMAStore.cs

```diff
@@ -1172,5 +1187,233 @@ private string RemoveindexerFromPropertyName(string str)
 
             return str;
         }
+
+        #region Inherit docs
+        private void BuildInheritanceForDocs(Type t)
+        {
+            var allInheritedMembersById = new Dictionary<string, List<string>>();
+            if (t.BaseTypes != null)
+            {
+                t.InheritanceChains = BuildInheritanceChain(t.Uid);
```

Comment:
do we need to rebuild the inheritance chain here?

Reply:
updated.